E2E-ORS-Sample-For-Java/src/main/java/ObjectReplicationMonitor.java

```diff
@@ -0,0 +1,297 @@
+        List<String> completedEventsList = checkReplication(sourceStorageAccountConnectionString, sourceContainerName, replicatedBlobList);
+
+        // Call method to check the contents of a replicated blob
+        checkContents(sourceStorageAccountConnectionString, sourceContainerName, destinationStorageAccountConnectionString, destinationContainerName, completedEventsList);
+
+        // Call method to archive replicated blobs
+        if (archiveMethod.equals("batch")) {
+            archiveReplicatedDataWithBatch(destinationStorageAccountConnectionString, destinationContainerName, completedEventsList);
+        }
+        else if (archiveMethod.equals("individual")) {
+            archiveReplicatedDataIndividually(destinationStorageAccountConnectionString, destinationContainerName, completedEventsList);
+        }
+        else {
+            System.out.println("\nNo archive method selected");
+        }
+    }
+
+
+    private static void setupSource(String connectionString, String containerName, String[] blobList) {
+        /*
+        This method is to upload the blobs to the source container
+         */
+
+        System.out.println("\nUploading blobs to the source container...");
+
+        // Creating blobserviceclient in order to make containerclient
+        BlobServiceClient sourceBlobServiceClient = new BlobServiceClientBuilder().connectionString(connectionString).buildClient();
+
+        // Get the container client
+        BlobContainerClient sourceContainerClient = sourceBlobServiceClient.getBlobContainerClient(containerName);
+        if (!sourceContainerClient.exists()) {
+            sourceContainerClient.create();
+        }
+
+        // Iterate through list of blobs
+        for (String b : blobList) {
+            // Get blob client
+            BlobClient sourceBlobClient = sourceContainerClient.getBlobClient(b);
+
+            // Upload the blob
+            String content = "Hello World!";
+            ByteArrayInputStream dataStream = new ByteArrayInputStream(content.getBytes());
+            sourceBlobClient.upload(dataStream, content.length(), true);
+        }
+    }
+
+
+    private static List<String> checkReplication(String connectionString, String containerName, String[] blobList) {
+        /*
+        This method is to check the replication status of the blobs.
+         */
+
+        // Creating blobserviceclient to access containerclient
+        BlobServiceClient blobServiceClient = new BlobServiceClientBuilder().connectionString(connectionString).buildClient();
+
+        // Get the container client to access blob client
+        BlobContainerClient blobContainerClient = blobServiceClient.getBlobContainerClient(containerName);
+
+        // Variables to keep track of and represent how many blobs have replicated
+        int completed = 0;
+        double ratio = 0;
+        int percentage = (int) (ratio * 100);
```

Comment:
I think you can just set the percentage as 0, since you have the ratio starting at 0 every time